### PR TITLE
[qosorch]: Add cir and pir parameters for scheduler

### PR DESCRIPTION
**Pull request by tengfei-astfs**
Signed-off-by: tengfei <tengfei@asterfusion.com>

**What I did**
   Add  cir, pir, cbs, pbs parameters for scheduler 
**Why I did it**
   SAI interface supports cir, pir, cbs, pbs parameters for scheduler, but these parameters were not in orchagent 
**How I verified it**
   I wrote these parameters to redis database by json, and then verified by redis-cli

### CODE REVIEW

File: orchagent/qosorch.h [hunk 1]
```diff
@@ -36,6 +36,11 @@ const string scheduler_algo_STRICT              = "STRICT";
 const string scheduler_weight_field_name        = "weight";
 const string scheduler_priority_field_name      = "priority";
 
+const string scheduler_min_bandwidth_rate_field_name      = "cir";
```
Comment (stcheng) [inline orchagent/qosorch.h]: could you align the equal signs?

Comment (tengfei-astfs) [reply orchagent/qosorch.h]: I hava updated "qosorch.h".

File: orchagent/qosorch.h [hunk 2]
```diff
@@ -36,6 +36,15 @@ const string scheduler_algo_STRICT              = "STRICT";
...
 const string scheduler_priority_field_name      = "priority";
 
+const string scheduler_min_bandwidth_rate_field_name
+                                                = "cir";
+const string scheduler_min_bandwidth_burst_rate_field_name  
+                                                = "cbs";
+const string scheduler_max_bandwidth_rate_field_name        
+                                                = "pir";
+const string scheduler_max_bandwidth_burst_rate_field_name  
+                                                = "pbs";
```
Comment (wendani) [inline orchagent/qosorch.h]: Can you put each string to one line? Also, can you add comments to what these acronyms stand for?

**Comment by stcheng** [comment]
retest this please

**Comment by wendani** [comment]
retest this please

**Comment by wendani** [comment]
retest this please

**Pull request merged**